### PR TITLE
Adcm 1580 add allure steps to ui tests

### PR DESCRIPTION
**Pull request by annademina**

### CODE REVIEW

File: tests/ui_tests/app/app.py [hunk 1]
```diff
@@ -95,19 +93,24 @@ def get_element(el):
             return WDW(self.driver, timer).until(condition(el))
         return get_element(locator)
 
+    @allure.step('Wait for element displayed')
```
Comment (dgusakov) [inline tests/ui_tests/app/app.py:96]: Can we add information about what element we are waiting for?

File: tests/ui_tests/app/configuration.py [hunk 2]
```diff
@@ -26,23 +28,13 @@ def __init__(self, driver, url=None):
...
     def assert_field_editable(self, field, editable=True):
-        """Check that we can edit specific field or not
-        :param field:
-        :param editable:
-        :return:
-        """
         field_editable = self.editable_element(field)
         assert field_editable == editable
 
+    @allure.step('Check that field content equal expected value: {expected_value}')
```
Comment (dgusakov) [inline tests/ui_tests/app/configuration.py:36]:
```suggestion
    @allure.step('Check that field content is equal to expected value: {expected_value}')
```

Information about field we are investigating we be also usefull

File: tests/ui_tests/app/configuration.py [hunk 3]
```diff
@@ -75,80 +63,49 @@ def assert_alerts_presented(self, field_type):
...
-        """
+    @allure.step('Check that expected text in form field: {expected_text}')
+    def assert_text_in_form_field_element(self, element: WebElement, expected_text: str):
         result = self._wait_text_element_in_element(element,
                                                     Common.mat_form_field,
                                                     text=expected_text)
         err_msg = "Expected text not presented: {}.".format(expected_text)
         assert result, err_msg
 
+    @allure.step('Get key value for map field')
```
Comment (dgusakov) [inline tests/ui_tests/app/configuration.py:95]: Do we really want to see this step in report?

File: tests/ui_tests/app/configuration.py [hunk 4]
```diff
@@ -75,80 +63,49 @@ def assert_alerts_presented(self, field_type):
...
-        """Get key value for map field
-
-        :param item_element:
-        :return:
-        """
         form_field = item_element.find_element(*ConfigurationLocators.map_key_field)
         inp = form_field.find_element(*Common.mat_input_element)
         return inp.get_attribute("value")
 
+    @allure.step('Get value for map field')
```
Comment (dgusakov) [inline tests/ui_tests/app/configuration.py:101]: Do we really want to see this step in report?

File: tests/ui_tests/app/configuration.py [hunk 5]
```diff
@@ -75,80 +63,49 @@ def assert_alerts_presented(self, field_type):
...
-        """Get value for map field
-
-        :param item_element:
-        :return:
-        """
         form_field = item_element.find_element(*ConfigurationLocators.map_value_field)
         inp = form_field.find_element(*Common.mat_input_element)
         return inp.get_attribute("value")
 
+    @allure.step('Get map field values')
```
Comment (dgusakov) [inline tests/ui_tests/app/configuration.py:107]: Do we really want to see this step in report?

File: tests/ui_tests/app/configuration.py [hunk 6]
```diff
@@ -190,12 +141,8 @@ def get_field_value_by_type(self, field_element, field_type):
         return current_value
 
     @staticmethod
-    def get_structure_values(field):
-        """Get structure values for field
-
-        :param field:
-        :return: list of dicts
-        """
+    @allure.step('Get structure values for field')
```
Comment (dgusakov) [inline tests/ui_tests/app/configuration.py:144]: Do we really want to see this step in report?

File: tests/ui_tests/app/configuration.py [hunk 7]
```diff
@@ -157,14 +114,8 @@ def get_map_field_config(self, map_field):
...
 
-    def get_field_value_by_type(self, field_element, field_type):
-        """Return field value from element by field type
-
-        :param field_element: WebElement
-        :param field_type: string with type
-        :return: field value, for numeric fields string will be converted
-        to field type.
-        """
+    @allure.step('Get field value from element by field type')
```
Comment (dgusakov) [inline tests/ui_tests/app/configuration.py:117]: Do we really want to see this step in report?

**Review by dgusakov — CHANGES_REQUESTED** [review]
Pls make sure that all step decorators are really worth being in report.

File: tests/ui_tests/app/app.py (hunk 1 above)
Comment (annademina) [reply tests/ui_tests/app/app.py:96]: need separate task for adding one more value to locators if it's necessary

**Review by vgorkavenko — APPROVED** [review]

**Review by skhomuti — APPROVED** [review]

**Review by dgusakov — APPROVED** [review]

**Pull request merged**